libr/anal/p/anal_x86_cs.c

```diff
@@ -1403,6 +1403,42 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 	}
 }
 
+static int parse_reg_name_mov(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
+	if (!reg) { return -1; }
+
+	switch (INSOP(reg_num).type) {
+	case X86_OP_REG:
+		reg->name = cs_reg_name (*handle, INSOP(reg_num).reg);
+		break;
+	default:
+		break;
+	}
+
+	return 0;
+}
+
+static int parse_reg_name_lea(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
+	if (!reg) { return -1; }
```

Comment:
To be honest it makes me mad. If i write
``` 
if (something) {
    ;
}
```
You ask me to change it to
```
if (something) {}
```
And if I write in one line, you ask to change it back :/